components/utilities/direction/index.jsx

```diff
@@ -0,0 +1,28 @@
+/* Copyright (c) 2015-present, salesforce.com, inc. All rights reserved */
+/* Licensed under BSD 3-Clause - see LICENSE.txt or git.io/sfdc-license */
```

Comment:
I think we might want the URL to be `components/utilities/UNSAFE_direction/index.jsx`. Otherwise you'd never see the unsafe part example for the examples.